backend/src/govsg/routes/govsg-campaign.routes.ts

```diff
@@ -133,7 +132,7 @@ router.get(
     [Segments.QUERY]: Joi.object({
       limit: Joi.number().integer().min(1).max(100).default(10),
       offset: Joi.number().integer().min(0).default(0),
-      status: Joi.string().valid(...Object.values(GovsgMessageStatus)),
+      search: Joi.string(),
```

Comment:
I have two questions regarding this:
1. Why did we previously validate `status` when it is not used in `listMessages`
2. Why do we not need it now?

Reply:
(1) A copy mistake.
(2) Cus I realised...